server/controllers/profile.controller.js

```diff
@@ -41,9 +47,11 @@ class ProfileController {
       username,
       bio,
       image,
-      articles: `${userActivity[0]} articles written.`,
-      // following: `You are following ${userActivity[1]} authors.`,
-      // followers: `You have ${userActivity[2]} followers.`
+      articlesWritten: `${userActivity[0].count} articles written.`,
+      articlesWrittenList: userActivity[0].rows,
+      articlesRead: `You have read ${userActivity[1].count} article(s).`,
+      articlesReadList: userActivity[1].rows,
+      followers: `You have ${userActivity[2].length} followers.`
```

Comment:
Same thing here.

Reply:
We will be showing just the figures for now. I included the list just in case there is need for it in the frontend.